package.json

```diff
@@ -173,6 +173,7 @@
     "**/@babel/preset-typescript": "^7.10.4",
     "**/@babel/runtime": "^7.10.2",
     "**/@babel/types": "^7.10.2",
+    "**/@emotion/styled": "^11.0.0",
```

Comment:
This seems unsafe. theme-ui depends on 10.x and for benchmarking purposes we shouldn't alter their package.

Reply:
This is the diff that we found that pinpointed the origin of the fail of the tests. We had the tests importing the ThemeContext of a wrong version. Any idea for a better solution? Maybe `@emotion/styled` should reexport the ThemeContext. But even then, we would need to test v11, not v10. Could it be that we miss the dependency in the root package.json?

Reply:
> Could the core of the problem be that we miss the dependency in the root package.json, (having it under docs isn't enough)?

If benchmark depends on it then it should be fine. There's likely an issue with our dependency tree and forcing a single version hides it.

Reply:
Should we add the dependency then on the root? Or leave as it is now?

Reply:
I've added the dependency on the root and removed the resolution and locally the tests passed. If the CI tests pass too I think we can resolve 👍